components/search/list-renderer/index.js

```diff
@@ -37,6 +37,7 @@ import RodentDonor from "./rodent-donor";
 import TechnicalSample from "./technical-sample";
 import Treatment from "./treatment";
 import User from "./user";
+import File from "./file";
```

Comment:
Not something you need to change, but I try to keep imports sorted by file name so it’s easy to find at a glance, for future PRs. That’s why imports for most files are broken up into sections (// node_modules, // components, etc.) so sorting by file name is easier (sections don’t apply to this file), and it’s easier to find something just by looking.